projects/kit/components/input-date/multi-date/input-date-multi.component.ts

```diff
@@ -0,0 +1,355 @@
+        }
+
+        return (
+            this.month ||
+            this.value[0] ||
+            tuiDateClamp(this.defaultActiveYearMonth, this.computedMin, this.computedMax)
+        );
+    }
+
+    get nativeValue(): string {
+        return this.nativeFocusableElement?.value ?? '';
+    }
+
+    set nativeValue(value: string) {
+        if (!this.nativeFocusableElement) {
+            return;
+        }
+
+        this.nativeFocusableElement.value = value.toString();
+    }
+
+    get canOpen(): boolean {
+        return this.interactive && !this.computedMobile;
+    }
+
+    getComputedFiller(filler: string): string {
+        return this.value.length ? '' : filler;
+    }
+
+    onIconClick(): void {
+        if (!this.computedMobile || !this.mobileCalendar) {
+            return;
+        }
+
+        this.dialogs
+            .open<readonly TuiDay[]>(
+                new PolymorpheusComponent(this.mobileCalendar, this.injector),
+                {
+                    size: 'fullscreen',
+                    closeable: false,
+                    data: {
+                        single: false,
+                        min: this.min,
+                        max: this.max,
+                        disabledItemHandler: this.disabledItemHandler,
+                    },
+                },
+            )
+            .pipe(takeUntil(this.destroy$))
+            .subscribe(value => {
+                this.internalValue = value;
+                this.value = value;
+            });
+    }
+
+    onEnter(search: string): void {
+        if (this.tagValidator(search)) {
+            this.internalValue = tuiImmutableUpdateInputDateMulti(
+                this.internalValue,
+                TuiDay.normalizeParse(search),
+            );
+
+            if (this.inputTag) {
+                this.inputTag.search = '';
+            }
+
+            this.done();
+        }
+    }
+
+    onValueChange(value: readonly TuiDay[]): void {
+        this.control?.updateValueAndValidity({emitEvent: false});
+
+        if (!value.length) {
+            this.onOpenChange(true);
+        }
+
+        this.internalValue = value;
+        this.value = value;
+    }
+
+    onDayClick(value: TuiDay): void {
+        this.internalValue = tuiImmutableUpdateInputDateMulti(this.internalValue, value);
+    }
+
+    done(): void {
+        this.value = this.internalValue;
+        this.open = false;
+    }
+
+    onMonthChange(month: TuiMonth): void {
+        this.month = month;
+    }
+
+    onOpenChange(open: boolean): void {
+        this.open = open;
+    }
+
+    onFocused(focused: boolean): void {
+        this.updateFocused(focused);
+    }
+
+    override setDisabledState(): void {
+        super.setDisabledState();
+        this.open = false;
+    }
+
+    override writeValue(value: readonly TuiDay[]): void {
+        this.internalValue = value;
+        super.writeValue(value);
+        this.nativeValue = value.toString();
```

Comment:
This seems wrong, why native value is set to stringified version of an entire array?